docs/products/postgresql/concepts/upgrade-failover.rst

```diff
@@ -57,3 +57,33 @@ During maintenance updates, cloud migrations, or plan changes, the below procedu
 
 .. Note::
     The old primary server is kept alive for a short period of time (minimum 60 seconds) with a TCP forwarding setup pointing to the new primary server allowing clients to connect before learning the new IP address.
+
+Recreation of replication slots
+-------------------------------
+
+In case of failover or controlled switchover of an Aiven for PostgreSQL service, the replication slots from the old primary server are automatically recreated in the new primary server.
+
+.. important::
+
+    The recreation of replication slots feature is enabled automatically and doesn't require restarting the nodes.
```

Comment:
Customer can't restart the node anyway, so I would remove this second part of the sentence.